### PR TITLE
mod upper leters for apiserver

### PR DESCRIPTION
**Pull request by chuntaochen**
What type of PR is this?

/kind cleanup

What this PR does / why we need it:
Fix some errors info start with capital letters in apiserver
xref:
https://gist.github.com/porjo/b331df0b3d7e33c30920

 Which issue(s) this PR fixes:

Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?

```release-note
 NONE
```

Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+
```
Comment (apelisse) [inline staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go:17]: Can you remove this extra change while you're at it?

File: plugin/pkg/admission/securitycontext/scdeny/admission.go [hunk 2]
```diff
@@ -79,21 +79,21 @@ func (p *Plugin) Validate(ctx context.Context, a admission.Attributes, o admissi
 	for _, v := range pod.Spec.InitContainers {
 		if v.SecurityContext != nil {
 			if v.SecurityContext.SELinuxOptions != nil {
-				return apierrors.NewForbidden(a.GetResource().GroupResource(), pod.Name, fmt.Errorf("SecurityContext.SELinuxOptions is forbidden"))
```
Comment (apelisse) [inline plugin/pkg/admission/securitycontext/scdeny/admission.go:82]: These four occurrences I don't know. They are referring to a field that is capitalized, I would probably leave them as-is

File: staging/src/k8s.io/apiserver/pkg/endpoints/filters/authn_audit.go [hunk 3]
```diff
@@ -80,7 +80,7 @@ func getAuthMethods(req *http.Request) string {
 	}
 
 	if len(authMethods) > 0 {
-		return fmt.Sprintf("Authentication failed, attempted: %s", strings.Join(authMethods, ", "))
+		return fmt.Sprintf("authentication failed, attempted: %s", strings.Join(authMethods, ", "))
```
Comment (sttts) [inline staging/src/k8s.io/apiserver/pkg/endpoints/filters/authn_audit.go:83]: is lower-case consistent here? What is this string used for?

See line below.

**Pull request closed without merging**